ui/raidboss/data/04-sb/eureka/eureka_hydatos.js

```diff
@@ -437,11 +347,7 @@
       condition: function(data) {
         return data.CanSleep();
       },
-      alertText: {
-        en: 'Sleep Centaur',
-        de: 'Zentaur einschläfern',
-        cn: '沉默人马',
-      },
+      response: Responses.sleep(),
```

Comment:
...this loses the target

Reply:
fixed in another pr (including of source into sleep/stun/interupt)